suites/suites/suites.yaml

```diff
@@ -658,6 +666,23 @@ handler_configurations:
       vsphere_resource_pool_name: 'system_tests'
       manager_server_template: 'systemtests-ubuntu-1404'
 
+  ###############################################################################
+
+  vcloud_handler_config:
+    inputs: inputs-vcloud.yaml
+    handler: vcloud_handler
+    env: vcloud
+    external: *vcloud_plugin_external
+    manager_blueprint: new/vcloud-manager-blueprint.yaml
+    properties: vcloud_handler_props
+    manager_blueprint_override: *vcloud_manager_blueprint_override
+    inputs_override:
+      vcloud_username: '{{system_test_vcloud_username}}'
+      vcloud_password: '{{system_test_vcloud_password}}'
+      vcloud_service_type: '{{system_test_vcloud_service_type}}'
```

Comment:
shouldn't `vcloud_service_type` and the two that follow it be defined in the `variables` section? are they private in any way to justify putting them in quickbuild?
(also, add the ones that are in quickbuild to the upper section where these are documented with a default `_`